### DISCUSSION TITLE
Math doesn’t seem to render correctly on the web page

### DISCUSSION THREAD

**Discussion by ayberkt** [standardml/twelf#46] [General]
When I look at the [main page](https://twelf.org/wiki/theorem-prover/), I see `<math>\texttt{plus}(N,0,N)</math>`and so on instead of the renderings.

**Comment by robsimmons** [comment]
Hi! When we moved twelf.org to an Astro site, the math was a bit of a casualty, and there's still some work to do updating old material. I've fixed this particular instance in #50, though there's plenty of other pages that still need to be updated.
